web-gui/src/app/tool-components/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -0,0 +1,27 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-confirmation-dialog',
+  templateUrl: './confirmation-dialog.component.html',
+})
+export class ConfirmationDialogComponent {
+  message: String = 'Are you sure?';
+  confirmButtonText = 'Yes';
+  cancelButtonText = 'Cancel';
+  constructor(
+    @Inject(MAT_DIALOG_DATA) private data: any,
+    private dialogRef: MatDialogRef<ConfirmationDialogComponent>) {
+    if (data) {
+      this.message = data.message || this.message;
+      if (data.buttonText) {
+        this.confirmButtonText = data.buttonText.ok || this.confirmButtonText;
+        this.cancelButtonText = data.buttonText.cancel || this.cancelButtonText;
+      }
+    }
+  }
+  onConfirmClick(): void {
+    this.dialogRef.close(true);
+  }
+}
+
```

Comment:
There is already a component for displaying a dialog (https://github.com/thm-mni-ii/feedbacksystem/blob/eda2d80b3d2667bd2681b5ce0d485fa1142474ec/web-gui/src/app/dialogs/confirm-dialog/confirm-dialog.component.ts). Why is a new one being created here? Shouldn't it just be possible to use this one?